app/routes/schema-browser/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { Link } from '@remix-run/react'
+
+export default function () {
+  return (
+    <>
+      <h1>Schema Browser</h1>
+      <p>
+        Browse the schema definitions for GCN Notices as distributed by GCN
+        Kafka. Navigate through the directory tree and select a schema for more
```

Comment:
Not sure how to explain it, but the IGWN JSON schema don't follow the unified schema, so we just need to make it clear that this is for other types, and link to the IGWN schema docs:  https://emfollow.docs.ligo.org/userguide/content.html#